### PR TITLE
Update the project after three or four years

### PR DESCRIPTION
**Pull request by reinout**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -16,8 +16,7 @@ Checkoutmanager works on linux, osx and windows.
 Starting to use it
 ------------------
 
-Starting is easy.  Just ``easy_install checkoutmanager`` and run
-``checkoutmanager``.
+Starting is easy.  Just ``pip checkoutmanager`` and run ``checkoutmanager``.
```
Comment (mauritsvanrees) [inline README.rst]: That should be pip **install** checkoutmanager.

Comment (reinout) [reply README.rst]: Oopsie!
Thanks, I've fixed it in 2.6.1. This just looks too ridiculous to leave as-is on pypi :-)

**Comment by rafaelbco** [comment]
I'm sorry to interrupt this family gathering. I'm here just to thank you for this awesome project which I've been using daily for many years. Thank you for keeping it up-to-date!

**Comment by reinout** [comment]
Thanks for the feedback, @rafaelbco.
Tell other people about this project :-)

**Pull request merged**